[2025-03-09T22:39:32.259+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-09T22:39:32.291+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_gcs_job.submit_spark_gcc_job manual__2025-03-09T22:20:07.532183+00:00 [queued]>
[2025-03-09T22:39:32.308+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_gcs_job.submit_spark_gcc_job manual__2025-03-09T22:20:07.532183+00:00 [queued]>
[2025-03-09T22:39:32.308+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 2
[2025-03-09T22:39:32.329+0000] {taskinstance.py:2889} INFO - Executing <Task(SparkSubmitOperator): submit_spark_gcc_job> on 2025-03-09 22:20:07.532183+00:00
[2025-03-09T22:39:32.341+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spark_gcs_job', 'submit_spark_gcc_job', 'manual__2025-03-09T22:20:07.532183+00:00', '--job-id', '425', '--raw', '--subdir', 'DAGS_FOLDER/gcs_conn_test.py', '--cfg-path', '/tmp/tmpu9guhlib']
[2025-03-09T22:39:32.343+0000] {standard_task_runner.py:105} INFO - Job 425: Subtask submit_spark_gcc_job
[2025-03-09T22:39:32.344+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-09T22:39:32.344+0000] {standard_task_runner.py:72} INFO - Started process 178 to run task
[2025-03-09T22:39:32.390+0000] {task_command.py:467} INFO - Running <TaskInstance: spark_gcs_job.submit_spark_gcc_job manual__2025-03-09T22:20:07.532183+00:00 [running]> on host 27f453beda85
[2025-03-09T22:39:32.489+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_gcs_job' AIRFLOW_CTX_TASK_ID='submit_spark_gcc_job' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T22:20:07.532183+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T22:20:07.532183+00:00'
[2025-03-09T22:39:32.490+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-09T22:39:32.532+0000] {base.py:84} INFO - Retrieving connection 'spark_default'
[2025-03-09T22:39:32.533+0000] {spark_submit.py:474} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem --conf spark.hadoop.fs.gs.auth.service.account.enable=true --conf spark.hadoop.fs.gs.auth.service.account.json.keyfile=/opt/***/keys/first-key.json --jars /opt/***/jars/gcs-connector-hadoop3-2.2.0.jar --total-executor-cores 2 --executor-memory 2g --name spark_gcs_job --verbose --deploy-mode client /opt/***/jobs/gcc_test_job.py
[2025-03-09T22:39:32.593+0000] {spark_submit.py:641} INFO - /home/***/.local/lib/python3.12/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2025-03-09T22:39:35.074+0000] {spark_submit.py:641} INFO - Using properties file: null
[2025-03-09T22:39:35.224+0000] {spark_submit.py:641} INFO - Parsed arguments:
[2025-03-09T22:39:35.225+0000] {spark_submit.py:641} INFO - master                  spark://spark:7077
[2025-03-09T22:39:35.225+0000] {spark_submit.py:641} INFO - remote                  null
[2025-03-09T22:39:35.225+0000] {spark_submit.py:641} INFO - deployMode              client
[2025-03-09T22:39:35.225+0000] {spark_submit.py:641} INFO - executorMemory          2g
[2025-03-09T22:39:35.225+0000] {spark_submit.py:641} INFO - executorCores           null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - totalExecutorCores      2
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - propertiesFile          null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - driverMemory            null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - driverCores             null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - driverExtraClassPath    null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - driverExtraLibraryPath  null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - driverExtraJavaOptions  null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - supervise               false
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - queue                   null
[2025-03-09T22:39:35.226+0000] {spark_submit.py:641} INFO - numExecutors            null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - files                   null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - pyFiles                 null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - archives                null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - mainClass               null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - primaryResource         file:/opt/***/jobs/gcc_test_job.py
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - name                    spark_gcs_job
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - childArgs               []
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - jars                    file:/opt/***/jars/gcs-connector-hadoop3-2.2.0.jar
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - packages                null
[2025-03-09T22:39:35.227+0000] {spark_submit.py:641} INFO - packagesExclusions      null
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - repositories            null
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - verbose                 true
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - Spark properties used, including those specified through
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - --conf and those from the properties file null:
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.auth.service.account.enable,true)
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.auth.service.account.json.keyfile,/opt/***/keys/first-key.json)
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.impl,com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem)
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:39:35.228+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:39:35.754+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-03-09T22:39:36.247+0000] {spark_submit.py:641} INFO - Main class:
[2025-03-09T22:39:36.247+0000] {spark_submit.py:641} INFO - org.apache.spark.deploy.PythonRunner
[2025-03-09T22:39:36.250+0000] {spark_submit.py:641} INFO - Arguments:
[2025-03-09T22:39:36.251+0000] {spark_submit.py:641} INFO - file:/opt/***/jobs/gcc_test_job.py
[2025-03-09T22:39:36.252+0000] {spark_submit.py:641} INFO - null
[2025-03-09T22:39:36.258+0000] {spark_submit.py:641} INFO - Spark config:
[2025-03-09T22:39:36.258+0000] {spark_submit.py:641} INFO - (spark.app.name,spark_gcs_job)
[2025-03-09T22:39:36.259+0000] {spark_submit.py:641} INFO - (spark.app.submitTime,1741559976173)
[2025-03-09T22:39:36.259+0000] {spark_submit.py:641} INFO - (spark.cores.max,2)
[2025-03-09T22:39:36.259+0000] {spark_submit.py:641} INFO - (spark.executor.memory,2g)
[2025-03-09T22:39:36.259+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.auth.service.account.enable,true)
[2025-03-09T22:39:36.260+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.auth.service.account.json.keyfile,/opt/***/keys/first-key.json)
[2025-03-09T22:39:36.260+0000] {spark_submit.py:641} INFO - (spark.hadoop.fs.gs.impl,com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem)
[2025-03-09T22:39:36.260+0000] {spark_submit.py:641} INFO - (spark.jars,file:///opt/***/jars/gcs-connector-hadoop3-2.2.0.jar)
[2025-03-09T22:39:36.260+0000] {spark_submit.py:641} INFO - (spark.master,spark://spark:7077)
[2025-03-09T22:39:36.260+0000] {spark_submit.py:641} INFO - (spark.repl.local.jars,file:///opt/***/jars/gcs-connector-hadoop3-2.2.0.jar)
[2025-03-09T22:39:36.261+0000] {spark_submit.py:641} INFO - (spark.submit.deployMode,client)
[2025-03-09T22:39:36.261+0000] {spark_submit.py:641} INFO - (spark.submit.pyFiles,)
[2025-03-09T22:39:36.262+0000] {spark_submit.py:641} INFO - Classpath elements:
[2025-03-09T22:39:36.262+0000] {spark_submit.py:641} INFO - file:///opt/***/jars/gcs-connector-hadoop3-2.2.0.jar
[2025-03-09T22:39:36.263+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:39:36.263+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:39:38.206+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkContext: Running Spark version 3.5.5
[2025-03-09T22:39:38.208+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkContext: OS info Linux, 5.15.167.4-microsoft-standard-WSL2, amd64
[2025-03-09T22:39:38.209+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkContext: Java version 17.0.14
[2025-03-09T22:39:38.261+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceUtils: ==============================================================
[2025-03-09T22:39:38.263+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-03-09T22:39:38.263+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceUtils: ==============================================================
[2025-03-09T22:39:38.264+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkContext: Submitted application: GCSConnectorTest
[2025-03-09T22:39:38.297+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 2048, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-03-09T22:39:38.314+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceProfile: Limiting resource is cpu
[2025-03-09T22:39:38.314+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-03-09T22:39:38.410+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SecurityManager: Changing view acls to: ***
[2025-03-09T22:39:38.411+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SecurityManager: Changing modify acls to: ***
[2025-03-09T22:39:38.411+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SecurityManager: Changing view acls groups to:
[2025-03-09T22:39:38.411+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SecurityManager: Changing modify acls groups to:
[2025-03-09T22:39:38.412+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2025-03-09T22:39:38.746+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO Utils: Successfully started service 'sparkDriver' on port 34989.
[2025-03-09T22:39:38.809+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkEnv: Registering MapOutputTracker
[2025-03-09T22:39:38.864+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkEnv: Registering BlockManagerMaster
[2025-03-09T22:39:38.896+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-03-09T22:39:38.896+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-03-09T22:39:38.905+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-03-09T22:39:38.952+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-d576dfc1-4979-44cb-98a8-151e6de23bae
[2025-03-09T22:39:38.975+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-03-09T22:39:38.993+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:38 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-03-09T22:39:39.206+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:39 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-03-09T22:39:39.326+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:39 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-03-09T22:39:39.377+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:39 INFO SparkContext: Added JAR file:///opt/***/jars/gcs-connector-hadoop3-2.2.0.jar at spark://27f453beda85:34989/jars/gcs-connector-hadoop3-2.2.0.jar with timestamp 1741559978188
[2025-03-09T22:39:39.506+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:39 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2025-03-09T22:39:41.614+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:41 WARN TransportClientFactory: DNS resolution failed for spark/<unresolved>:7077 took 2076 ms
[2025-03-09T22:39:41.650+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:41 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2025-03-09T22:39:41.651+0000] {spark_submit.py:641} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2025-03-09T22:39:41.651+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2025-03-09T22:39:41.651+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2025-03-09T22:39:41.652+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2025-03-09T22:39:41.652+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2025-03-09T22:39:41.652+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2025-03-09T22:39:41.653+0000] {spark_submit.py:641} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2025-03-09T22:39:41.653+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2025-03-09T22:39:41.654+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2025-03-09T22:39:41.654+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-03-09T22:39:41.654+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - Caused by: java.io.IOException: Failed to connect to spark/<unresolved>:7077
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2025-03-09T22:39:41.655+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - ... 4 more
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - Caused by: java.net.UnknownHostException: spark
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2025-03-09T22:39:41.656+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2025-03-09T22:39:41.657+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2025-03-09T22:39:41.658+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2025-03-09T22:39:41.659+0000] {spark_submit.py:641} INFO - ... 1 more
[2025-03-09T22:39:59.508+0000] {spark_submit.py:641} INFO - 25/03/09 22:39:59 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2025-03-09T22:40:01.545+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:01 WARN TransportClientFactory: DNS resolution failed for spark/<unresolved>:7077 took 2035 ms
[2025-03-09T22:40:01.547+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:01 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2025-03-09T22:40:01.547+0000] {spark_submit.py:641} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:40:01.548+0000] {spark_submit.py:641} INFO - Caused by: java.io.IOException: Failed to connect to spark/<unresolved>:7077
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - ... 4 more
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - Caused by: java.net.UnknownHostException: spark
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2025-03-09T22:40:01.549+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2025-03-09T22:40:01.550+0000] {spark_submit.py:641} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2025-03-09T22:40:01.551+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2025-03-09T22:40:01.552+0000] {spark_submit.py:641} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2025-03-09T22:40:01.553+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2025-03-09T22:40:01.553+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2025-03-09T22:40:01.553+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2025-03-09T22:40:01.553+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2025-03-09T22:40:01.553+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2025-03-09T22:40:01.554+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2025-03-09T22:40:01.555+0000] {spark_submit.py:641} INFO - ... 1 more
[2025-03-09T22:40:19.509+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:19 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2025-03-09T22:40:21.548+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:21 WARN TransportClientFactory: DNS resolution failed for spark/<unresolved>:7077 took 2037 ms
[2025-03-09T22:40:21.551+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:21 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2025-03-09T22:40:21.551+0000] {spark_submit.py:641} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2025-03-09T22:40:21.552+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2025-03-09T22:40:21.552+0000] {spark_submit.py:641} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2025-03-09T22:40:21.552+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2025-03-09T22:40:21.553+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2025-03-09T22:40:21.553+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2025-03-09T22:40:21.553+0000] {spark_submit.py:641} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2025-03-09T22:40:21.553+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2025-03-09T22:40:21.554+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2025-03-09T22:40:21.554+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-03-09T22:40:21.554+0000] {spark_submit.py:641} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-03-09T22:40:21.554+0000] {spark_submit.py:641} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:40:21.554+0000] {spark_submit.py:641} INFO - Caused by: java.io.IOException: Failed to connect to spark/<unresolved>:7077
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2025-03-09T22:40:21.555+0000] {spark_submit.py:641} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - ... 4 more
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - Caused by: java.net.UnknownHostException: spark
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2025-03-09T22:40:21.556+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2025-03-09T22:40:21.557+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2025-03-09T22:40:21.558+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2025-03-09T22:40:21.559+0000] {spark_submit.py:641} INFO - ... 1 more
[2025-03-09T22:40:39.511+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2025-03-09T22:40:39.512+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2025-03-09T22:40:39.527+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 38909.
[2025-03-09T22:40:39.527+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO NettyBlockTransferService: Server created on 27f453beda85:38909
[2025-03-09T22:40:39.530+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-03-09T22:40:39.552+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 27f453beda85, 38909, None)
[2025-03-09T22:40:39.559+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO BlockManagerMasterEndpoint: Registering block manager 27f453beda85:38909 with 434.4 MiB RAM, BlockManagerId(driver, 27f453beda85, 38909, None)
[2025-03-09T22:40:39.566+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 27f453beda85, 38909, None)
[2025-03-09T22:40:39.571+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 27f453beda85, 38909, None)
[2025-03-09T22:40:39.993+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:39 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-03-09T22:40:40.392+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-03-09T22:40:40.400+0000] {spark_submit.py:641} INFO - Traceback (most recent call last):
[2025-03-09T22:40:40.400+0000] {spark_submit.py:641} INFO - File "/opt/***/jobs/gcc_test_job.py", line 5, in <module>
[2025-03-09T22:40:40.401+0000] {spark_submit.py:641} INFO - .getOrCreate()
[2025-03-09T22:40:40.401+0000] {spark_submit.py:641} INFO - ^^^^^^^^^^^^^
[2025-03-09T22:40:40.401+0000] {spark_submit.py:641} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 500, in getOrCreate
[2025-03-09T22:40:40.401+0000] {spark_submit.py:641} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 589, in __init__
[2025-03-09T22:40:40.401+0000] {spark_submit.py:641} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2025-03-09T22:40:40.402+0000] {spark_submit.py:641} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2025-03-09T22:40:40.404+0000] {spark_submit.py:641} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.sql.SparkSession.
[2025-03-09T22:40:40.404+0000] {spark_submit.py:641} INFO - : java.lang.IllegalStateException: Cannot call methods on a stopped SparkContext.
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - This stopped SparkContext was created at:
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2025-03-09T22:40:40.405+0000] {spark_submit.py:641} INFO - java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.Gateway.invoke(Gateway.java:238)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - The currently active SparkContext was created at:
[2025-03-09T22:40:40.406+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - py4j.Gateway.invoke(Gateway.java:238)
[2025-03-09T22:40:40.407+0000] {spark_submit.py:641} INFO - py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at org.apache.spark.SparkContext.assertNotStopped(SparkContext.scala:122)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at org.apache.spark.sql.SparkSession.<init>(SparkSession.scala:113)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at org.apache.spark.sql.SparkSession.<init>(SparkSession.scala:106)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2025-03-09T22:40:40.408+0000] {spark_submit.py:641} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-03-09T22:40:40.409+0000] {spark_submit.py:641} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-03-09T22:40:40.410+0000] {spark_submit.py:641} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-03-09T22:40:40.410+0000] {spark_submit.py:641} INFO - 
[2025-03-09T22:40:40.422+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO SparkUI: Stopped Spark web UI at http://27f453beda85:4040
[2025-03-09T22:40:40.426+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO StandaloneSchedulerBackend: Shutting down all executors
[2025-03-09T22:40:40.430+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2025-03-09T22:40:40.439+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2025-03-09T22:40:40.453+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-03-09T22:40:40.491+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO MemoryStore: MemoryStore cleared
[2025-03-09T22:40:40.492+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO BlockManager: BlockManager stopped
[2025-03-09T22:40:40.492+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO ShutdownHookManager: Shutdown hook called
[2025-03-09T22:40:40.492+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-fce39bdf-a11e-4ee1-9a69-0d94a67037c6/pyspark-5410c4bf-b740-4314-9462-420814cd978c
[2025-03-09T22:40:40.506+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-03-09T22:40:40.507+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-03-09T22:40:40.515+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-fce39bdf-a11e-4ee1-9a69-0d94a67037c6
[2025-03-09T22:40:40.520+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO SparkContext: Successfully stopped SparkContext
[2025-03-09T22:40:40.524+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-00371c0f-0cc8-44c0-8c34-a9e556212d8c
[2025-03-09T22:40:40.527+0000] {spark_submit.py:641} INFO - 25/03/09 22:40:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-fce39bdf-a11e-4ee1-9a69-0d94a67037c6/userFiles-64d25ff3-d6a3-401a-a2a5-985ed3d17f2a
[2025-03-09T22:40:40.585+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 177, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 567, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem --conf spark.hadoop.fs.gs.auth.service.account.enable=true --conf spark.hadoop.fs.gs.auth.service.account.json.keyfile=/opt/***/keys/first-key.json --jars /opt/***/jars/gcs-connector-hadoop3-2.2.0.jar --total-executor-cores 2 --executor-memory 2g --name spark_gcs_job --verbose --deploy-mode client /opt/***/jobs/gcc_test_job.py. Error code is: 1.
[2025-03-09T22:40:40.593+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=spark_gcs_job, task_id=submit_spark_gcc_job, run_id=manual__2025-03-09T22:20:07.532183+00:00, execution_date=20250309T222007, start_date=20250309T223932, end_date=20250309T224040
[2025-03-09T22:40:40.622+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-09T22:40:40.622+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 425 for task submit_spark_gcc_job (Cannot execute: spark-submit --master spark://spark:7077 --conf spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem --conf spark.hadoop.fs.gs.auth.service.account.enable=true --conf spark.hadoop.fs.gs.auth.service.account.json.keyfile=/opt/***/keys/first-key.json --jars /opt/***/jars/gcs-connector-hadoop3-2.2.0.jar --total-executor-cores 2 --executor-memory 2g --name spark_gcs_job --verbose --deploy-mode client /opt/***/jobs/gcc_test_job.py. Error code is: 1.; 178)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 177, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 567, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem --conf spark.hadoop.fs.gs.auth.service.account.enable=true --conf spark.hadoop.fs.gs.auth.service.account.json.keyfile=/opt/***/keys/first-key.json --jars /opt/***/jars/gcs-connector-hadoop3-2.2.0.jar --total-executor-cores 2 --executor-memory 2g --name spark_gcs_job --verbose --deploy-mode client /opt/***/jobs/gcc_test_job.py. Error code is: 1.
[2025-03-09T22:40:40.665+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-03-09T22:40:40.680+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-09T22:40:40.684+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
